### DISCUSSION TITLE
"Bad word" matching

### DISCUSSION THREAD

**Discussion by lapftm** [Security-Onion-Solutions/securityonion#2603] [Unsupported Versions]
Getting "Bad word" matching as one of the alerts, Is this something that can be ignored and tuned out or is there something i'm not seeing in the Message or log output that i should be paying particular attention to?

Thanks for your time
lapftm.

**Comment by weslambert** [comment]
Hi @lapftm ,

You can add local rules here: https://docs.securityonion.net/en/latest/wazuh.html?highlight=wazuh#tuning-rules.

Alternatively, you can adjust `log_alert_level` (level at which Wazuh generates alerts) in `/nsm/wazuh/etc/ossec.conf` .

See the classification here: https://documentation.wazuh.com/3.13/user-manual/ruleset/rules-classification.html

Thanks,
Wes
